src/driver/sqlserver/SqlServerDriver.ts

```diff
@@ -310,30 +310,34 @@ export class SqlServerDriver implements Driver {
         if (!parameters || !Object.keys(parameters).length)
             return [sql, escapedParameters];
 
+        const queryPlaceholders: ObjectLiteral = {};
+        Object.keys(nativeParameters).map((key, index) => { queryPlaceholders[key] = "$" + index; });
```

Comment:
Why is this one different?

the others are `index + 1` but this one is just `index`.  Is it because they're 1-based indexing and this is 0-based indexing for parameters?